bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/util/BLangConstants.java

```diff
@@ -72,6 +72,7 @@
     public static final String USER_REPO_SRC_DIRNAME = "src";
     public static final String USER_REPO_OBJ_DIRNAME = "obj";
     public static final String USER_REPO_METADATA_DIRNAME = "metadata";
+    public static final String BAL_MAX_POOL_SIZE_ENV = "BALLERINA_MAX_POOL_SIZE";
```

Comment:
Shall we change this variable name to be consistent with other variables in this file?